packages/web-api/src/controllers/post-scans-controller.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { System } from 'common';
+import { ScanRunRequest } from '../api-contracts/scan-run-request';
+import { ScanRunResponse } from '../api-contracts/scan-run-response';
+import { ApiController } from './api-controller';
+
+export class PostScansController extends ApiController {
```

Comment:
>PostScansController [](start = 13, length = 19)

may be it might be better to have ScannController(to submit & receive scan info) & ReportController (to download report)

Reply:
Merge to single scan controller